Wemos_Modules/Example_Module/Example_Module.ino

```diff
@@ -134,11 +134,11 @@ unsigned int readDigitalInputs()
 }
 
 /* Set PCA9554 outputs (DIO4-DIO7) */
-void setDigitalOutput(int pin, bool state)
+void setDigitalOutput(byte digitalBuffer)
```

Comment:
Great work on this, I will add this to main ASAP.

Reply:
The commit adding this contains some other code. This will fix part of issue #6. Do you want to fix this or should I go ahead and fix it?